### ISSUE TITLE
Remove previous instance of trained model when re-training

### ISSUE THREAD

**Issue by TarekHC** [cta-observatory/iact_event_types#32]
I discovered today that when you train a model that has been already trained with a different input datasets (which may have a different energy binning) the previous instance of the trained model is not deleted. This gives errors in the later stages of the analysis (e.g. will extract all energy bins, from both instances).

**Comment by orelgueta** [comment]
Yes, I am aware of this and it annoyed me to no end until now. Never got around to fixing it =)
Will try to do it soon.

**Comment by TarekHC** [comment]
Hi Orel 

No rush at all! I just stumbled with this problem and lost some time until I realized about it... If you are aware of it, then another thing in our to-do list...
